docs/operating-scylla/procedures/cluster-management/replace-dead-node.rst

```diff
@@ -189,11 +189,11 @@ In this case, the node's data will be cleaned after restart. To remedy this, you
 
       sudo sed -e '/auto_bootstrap:.*/s/False/True/g' -i /etc/scylla/scylla.yaml
 
-#. Run the following command, replacing 172.30.0.186 with the listen_address / rpc_address of the node that you are restarting:
+#. Run the following command to replace the Host ID of the node that you are restarting:
```

Comment:
The previous statement instructed the reader to take the command below, modify the command by replacing the IP provided here with the IP of the node that they want to restart, and then run the command.

This statement tells the reader that the command replaces the Host ID of the node that the reader is restarting... replaces it with what? This doesn't make much sense. And running the command verbatim is obviously an error - we don't want the reader to run the following command, actually (we want them to run a modified command).

I think we should keep the format of the statement as it was and write something like:
"Run the following command, replacing 87a2b646-c968-4108-a38b-f8468b8b7360 with the Host ID of the node that you are restarting"

Reply:
yeah, that makes sense.

Reply:
I'm actually not happy with the phrase you suggested since the action is not about replacing the Host ID, it's about replacing the node previously known by that Host ID.

How about:
```
#. Run the following command to replace the node known by the previous Host ID of the node you are restarting, 87a2b646-c968-4108-a38b-f8468b8b7360, by the restarted instance, that will be assigned a new, random, Host ID:
```

Reply:
> action is not about replacing the Host ID

The command itself is not about replacing the Host ID, but the reader of the documentation must first replace the Host ID in the command with the Host ID of their node. That's what the "replacing 87a2b646-c968-4108-a38b-f8468b8b7360 with..." is about.

So the statement is a succint wait of saying:
1. In the command below, replace "87a2b646-c968-4108-a38b-f8468b8b7360" with the Host ID of the node that you are restarting
2. Then execute the command below.